### PR TITLE
test: Enable audit so we can properly test applying fixes

### PR DESCRIPTION
**Pull request by petervo**
This works on rhel as well.

### CODE REVIEW

File: test/verify/check-setroubleshoot [hunk 1]
```diff
@@ -134,17 +134,8 @@ class TestSelinux(MachineCase):
...
         # the fix should be applied
         b.wait_in_text(row_selector, "Solution applied successfully:")
-        # b.wait_in_text(row_selector, "Successfully ran /sbin/restorecon")
-
-        # HACK: we need updated test images for delete to work
-        # dismiss the alert
-        #b.click(row_selector + " button.pficon-delete")
-        #b.wait_not_present(row_selector)
-
-        #b.wait_in_text("body", "No SELinux alerts.")
```
Comment (martinpitt) [inline test/verify/check-setroubleshoot]: So dismissing an alert still doesn't work? That seems rather unrelated to enabling audit to spot them in the first place.

I realize that this code is already commented out in master, but would this work again?

Comment (martinpitt) [reply test/verify/check-setroubleshoot]: I tested that on Fedora-26. Lines 140, 144, and 145 do work again, but the very last line 147 doesn't, as apparently there is a second alert still:

testselinux-testtroubleshootalerts-fedora-26-127 0 0 2-2201-fail

I think that's moderately legitimate, so let's drop line 147 for good, and re-enable the others?

Comment (petervo) [reply test/verify/check-setroubleshoot]: I'm not sure they work on rhel, but i'll push a change to check
